charter.md

```diff
@@ -25,6 +25,8 @@ These metrics are expected to be comprised of:
 
 The team is responsible for maintaining [Interop 2021](https://wpt.fyi/interop-2021), [2022](https://wpt.fyi/interop-2022), [2023](https://wpt.fyi/interop-2023), and defining future interoperability efforts and metrics.
 
+If the Team is unable reach consensus to publish any new metrics, and has been unable to reach consensus for three months, the Team's functions revert to the Core Team and this charter is annulled.
```

Comment:
```suggestion
If the Team is unable to reach consensus, and therefore unable to progress in these areas, the Team's functions revert to the core team, and this charter is annulled.
```

Reply:
But what counts as "progress"? If the team is succeeding in maintaining the existing metrics, but failing at defining future interoperability efforts and metrics, is that still making "progress"?

Like, I think your argument is that it isn't, but I don't think that's clear.

Reply:
Ping @jgraham

Reply:
I think the answer is "I don't know, and therefore I don't want the charter to take a strong position on this". I think in practice any situation where the team was unable to reach agreement for an extended period of time would cause alarm bells to ring, and change the situation in a way that would either cause the core team to intervene or cause participants to leave the project in a way that would undermine its credibility in the wider community. 

In particular the restriction to "publish new metrics" wouldn't make sense in the hypothetical situation where we replaced annual metrics with a continuously updated "living metric". It's not clear to me if such a situation would be precluded by your proposed wording (would it be possible to reach consensus to _not_ publish new metrics?) or whether, if it was not, the proposed wording would cover a situation where we couldn't agree on how to _update_ an existing metric (vs publish a new one).

Reply:
Could everyone live with either saying nothing, or merely noting that the WPT core team has the power to dissolve the team / annul the charter? It sounds quite ominous, and is true even if we don't write it down, so my weak preference would be to say nothing.

Reply:
I think saying nothing is fine to me?

Reply:
Now that you've brought it up, I think some statement of the fact that the interop group (process-wise) ultimately answers to the wpt core team would be useful. That could be something on the wpt side that explains how sub-groups work.

Reply:
In general terms, I think @jgraham is right that we should put that in some Core Team document; prior to having such a thing, I'm happy to add a statement of fact to the Interop charter.

@jgraham, do you have any specific suggested text here?

Reply:
Having discussed this in the meeting, sounds like we have rough consensus to say nothing here. Going to resolve this thread, then.